elasticsearch/test/client-nodes-resources.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
No these are test resources when developing/testing some things in the top level azuredeploy.json template, they are essentially mocks that are generated from the referenced templates and deploy no resources.  The top level azuredeploy.json template can be changed to use the /test base path, and then you run a deployment you can test the parameter manipulation logic.  This makes editing the top level template much easier and faster.